pkg/rtc/mediatrackreceiver.go

```diff
@@ -960,6 +996,16 @@ func (t *MediaTrackReceiver) PrimaryReceiver() sfu.TrackReceiver {
 	return receivers[0].TrackReceiver
 }
 
+func (t *MediaTrackReceiver) ActiveReceiver() sfu.TrackReceiver {
+	for _, r := range t.loadReceivers() {
+		if r.IsRegressed() {
+			return r.TrackReceiver
+		}
+	}
+
+	return t.PrimaryReceiver()
+}
```

Comment:
Adding this method to use the regressed receiver for things like connection quality and audio level (in case we do simulcast codecs for audio at some point)